### ISSUE TITLE
GateSeal expiration alert hadn't been fired timely

### ISSUE THREAD

**Issue by TheDZhon** [lidofinance/alerting-forta#514]
## tl;dr

GateSeal expiration alert should have been fired three days ago because the contract's expiration date is set to **May, 1st** and the expiration threshold for the alert is four weeks (28 days).

## Description

The currently deployed GateSeal [`instance`]() has an expiration timestamp set at `1714521600`, i.e., **25** days away.

However, the [following code](https://github.com/lidofinance/alerting-forta/blob/f89bd6174771298090965240106f9c318a9f310c/ethereum-steth/src/services/gate-seal/GateSeal.srv.ts#L215-L228) should have fired an alert three days ago:
```ts
const GATE_SEAL_EXPIRY_THRESHOLD = ONE_MONTH
const GATE_SEAL_EXPIRY_TRIGGER_EVERY = ONE_WEEK
...
    } else if (
      currentBlockTimestamp - this.cache.getLastExpiryGateSealAlertTimestamp() >
      GATE_SEAL_EXPIRY_TRIGGER_EVERY
    ) {
      if (Number(expiryTimestamp) - currentBlockTimestamp <= GATE_SEAL_EXPIRY_THRESHOLD) {
        out.push(
          Finding.fromObject({
            name: '⚠️ GateSeal: is about to be expired',
            description: `GateSeal address: ${etherscanAddress(this.gateSealAddress)}\nExpiry date ${new Date(
              String(expiryTimestamp),
            )}`,
            alertId: 'GATE-SEAL-IS-ABOUT-TO-BE-EXPIRED',
            severity: FindingSeverity.Medium,
            type: FindingType.Degraded,
          }),
        )
        this.cache.setLastExpiryGateSealAlertTimestamp(currentBlockTimestamp)
      }
    }
```

Unluckily, it hadn't been fired due to the unknown reason _(edited)_

### References
- https://docs.lido.fi/deployed-contracts/
- https://research.lido.fi/t/renew-gateseal-for-the-withdrawal-queue-and-validator-exit-bus-oracle/7081

### How to reproduce

Given the expiration timestamp value is `1714521600`, substracting `GATE_SEAL_EXPIRY_THRESHOLD` (which is `60 * 60 * 24 * 7 * 4 = 2419200` gives us `1714521600 - 2419200 = 1712102400`, using the following API to fetch the block just after the threshold value:

https://api.findblock.xyz/v1/chain/1/block/after/1712102400
can conclude that the alert should have fired in the block [`19571532`](https://etherscan.io/block/19571532)

**Comment by sergeyWh1te** [comment]
PR - https://github.com/lidofinance/alerting-forta/pull/515
